site-src/implementations.md

```diff
@@ -0,0 +1,83 @@
+# Implementations
+
+This document tracks downstream implementations of Gateway API and provides status and resource references for them.
+
+Implementors of Gateway API are encouraged to update this document with status information about their implementations, the versions they cover, and documentation to help users get started.
+
+## Implementation Status
+
+- [Contour][1] (alpha)
```

Comment:
can we make this a table?

```
| Implementation | Status |
| ---- | ---- | 
etc
```

Reply:
Also, if we can put the version implemented (instead of just Alpha)
that would be useful for users

Reply:
e.g. Supports `v1alpha1`

Reply:
When this was in draft it started out as a table, see:

https://github.com/kubernetes-sigs/gateway-api/pull/565#discussion_r583944465

However @robscott had particularly requested that I change it to the format you see here.

If Rob is OK with changing it back to a table but using the format you suggest above, that sounds good to me. Just want to check with him.

Reply:
This is a minor thing and we can change in a later diff. My thinking was that table is easier to deal with more info (say which versions are supported)

Reply:
I also think a table would be a bit cleaner (though we don't have to block this PR). The original table had the vendors as columns instead of rows which doesn't leave much room for written content. The following format would allow us to take the existing written content we have in the page and surface it through a table.

```
| Implementation | Status |
| [Project/Product](link) | Explanation of progress towards support and/or support levels (which versions) | 
| [Project/Product](link) | Explanation of progress towards support and/or support levels (which versions) | 
etc
```

I think support is a gray area right now since we are early in the spec so it's prob too early for a clean cut version/checkmark box.